forem/forem
# [15 min fix] Updated husky package and configuration/pre-commit hook

## What type of PR is this? (check all applicable)


## Description

We've been ignoring updates to the husky package since they introduced breaking changes. I finally spent a small amount of time updating to the new version.

## Related Tickets & Documents

## QA Instructions, Screenshots, Recordings

1. The only thing to test is to grab this branch locally, change any file that lint-staged runs against, e.g. *.md, *.js etc. and ensure the husky pre-commit hook runs.

For example

```bash
forem on  nickytonline/husky-breaking-changes-upgrade [$+] is 📦 v1.0.0 via  v14.17.3 via 💎 v2.7.2 
❯ g ci "test"
✔ Preparing...
✔ Running tasks...
✔ Applying modifications...
✔ Cleaning up...
[nickytonline/husky-breaking-changes-upgrade 3c1af6b06] test
 1 file changed, 1 insertion(+)
```

### UI accessibility concerns?

N/A

## Added/updated tests?

- [x] No, and this is why: N/A

## [Forem core team only] How will this change be communicated?

_Will this PR introduce a change that impacts Forem members or creators, the
development process, or any of our internal teams? If so, please note how you
will share this change with the people who need to know about it._

      [Admin Guide](https://admin.forem.com/), or
      [Storybook](https://storybook.forem.com/) (for Crayons components)
      or in a [forem.dev](http://forem.dev) post
- [x] This change does not need to be communicated, and this is why not: This change will be transparent to contributors and core team members.

## [optional] Are there any post deployment tasks we need to perform?

## [optional] What gif best describes this PR or how it makes you feel?

![A husky dancing in a winter coat with pants on](https://media.giphy.com/media/AhC404o1telGIDToPC/giphy.gif)
